homeassistant/components/yeelight/__init__.py

```diff
@@ -234,8 +233,13 @@ def available(self):
 
     @property
     def model(self):
-        """Return configured device model."""
-        return self._model
+        """Return configured device model.
+
+        Model method returns declared model, and if not, it tries to get it from
```

Comment:
This PR removes the ability to declare the model, right?

Also, this property access does not do any IO anymore at this point(?), so the rest of the docstring should also be updated.

Reply:
Model still can be forced: https://gitlab.com/stavros/python-yeelight/-/blob/master/yeelight/main.py#L344

Reply:
Updated comment

Reply:
It is still confusing; is it worth describing all the information contained in the docstring at all? `Return configured/autodetected device model` would be simple enough.